lcu_integration.py

```diff
@@ -62,64 +95,49 @@ def connect_to_lcu(self, wait_for_availability: bool = False) -> bool:
         process_arguments = _get_lcu_commandline_arguments(lcu_process)
         self.install_directory = process_arguments["install-directory"]
         self._url = f"https://127.0.0.1:{process_arguments['app-port']}"
-        _auth_key = process_arguments['remoting-auth-token']
+        _auth_key = process_arguments["remoting-auth-token"]
 
         if self._session is not None:
-            logger.debug(
-                "LCU session already existed, closing it just to be safe."
-            )
+            logger.debug("LCU session already existed, closing it just to be safe.")
             self._session.close()
 
         self._session = requests.Session()
         self._session.auth = ("riot", _auth_key)
-        self._session.headers.update({
-            "Content-Type": "application/json",
-            "Accept": "application/json",
-        })
-        # TODO Do proper SSL integration
+        self._session.headers.update(
```

Comment:
nit: Hmm, I hadn't really considered this rule but I do find this one a bit.. excessive 🤔

Reply:
(I assume you selected the wrong line to comment on and meant to comment on the pylint at the TODO)
Yeah I agree, I think TODOs or FIXMEs in code are perfectly acceptable, as long as they're not forgotten completely :D 
I'll remove the rule in pylint.

Reply:
I meant to highlight the block but only selected the line -- lines 106-111 looks a little funky / "over spread out" (moving from `({` on one line to `(` and `{` on separate lines)

Not a strong feeling though!

I'll respond separately about the FIXME!